src/System.IO.Pipelines/project.json

```diff
@@ -14,7 +14,13 @@
   },
   "buildOptions": {
     "allowUnsafe": true,
-    "keyFile": "../../tools/Key.snk"
+    "keyFile": "../../tools/Key.snk",
+    "define": [
```

Comment:
Very useful to be able to disable  them, makes tests run 1000x faster. Maybe we need to move these settings to settings class but this suffices for now.